fish-rust/src/env.rs

```diff
@@ -0,0 +1,37 @@
+/// Flags that may be passed as the 'mode' in env_stack_t::set() / environment_t::get().
+pub mod flags {
+    use autocxx::c_int;
```

Comment:
I wonder what's the value of this `flags` module.
Constants like ENV_DEFAULT are intended to be used by their unqualified name.
I guess it does make our import statements a bit more transparent, and it's an obvious place for the doc comment.
We might want to apply this pattern to other flags. 

I'll change the // comment it to a doc comment.

Reply:
Yeah, there were another set of constants in this file, It seemed better to have separate mod for them. I think if we move from constant to enum or something else later than that will help with namespacing and docs

Reply:
yeah if both have the same ENV prefix that's another justification for the namespace